src/libs/ajax.js

```diff
@@ -11,6 +11,19 @@ import axios from 'axios';
 import {DataUseTranslation} from './dataUseTranslation';
 import {isFileEmpty} from './utils';
 
+//define axios interceptor
+//to log out user and redirect to home when response has 401 status
+//return responses with statuses in the 200s and reject the rest
+axios.interceptors.response.use(function (response) {
+  return response;
+}, function (error) {
+  if (error.status === 401) {
```

Comment:
To get the status on an error, you'll need to do ```error.response.status```

Reply:
It's specific to axios, which is probably why you don't handle errors the same way with fetchOK and fetchAny.